meta-freescale-3rdparty-mbl/recipes-bsp/atf/atf-imx6ul-pico-mbl_git.bb

```diff
@@ -2,15 +2,8 @@
 #
 # SPDX-License-Identifier: MIT
 
-inherit mbl-artifact-names
-require recipes-bsp/atf/atf.inc
+require atf-imx7-mbl_git.inc
```

Comment:
Should we rename the `atf-imx7-mbl_git.inc` file to `atf-imx-mbl_git.inc` as soon as it is now being used by imx7 and imx6 platforms?

Reply:
atf-imx6-imx7-mbl_git.inc since it is not used on imx8